packages/connect-agreement/src/thegraph/queries/index.ts

```diff
@@ -1,7 +1,7 @@
 import gql from 'graphql-tag'
 
 export const GET_AGREEMENT = (type: string) => gql`
-  ${type} Agreement($agreement: String!) {
+  query Agreement($agreement: String!) {
```

Comment:
Hmm, why are we changing the `type` to query? I guess it renders the type param useless, right?

Reply:
> Hmm, why are we changing the type to query?

The GraphQL endpoint errors if we try to perform a query with `subscription` in the query itself.

>  I guess it renders the type param useless, right?

I was trying to keep things as similar as possible to avoid having to ask connectors to adapt… but we will have to do it anyway. Thanks for pointing this out, let me check again to see if we can change the query from the object generated by `gql` so that connectors using `GraphQLWrapper` wouldn’t have to change anything. :+1:

Reply:
Done in https://github.com/aragon/connect/pull/203/commits/5a44409e03580b7d82280783370feeff6d6fd91d